### PR TITLE
update instructor bios

### LINKED ISSUE #42: Update Instructors info
Update landing page instructor data

### PR DESCRIPTION
**Pull request by brandonapol**
Closes #42

### CODE REVIEW

**Comment by brandonapol** [comment]
The second section of the landing page has two identical paragraphs, I used a component but I think I am going to migrate it into one component with two parts so I can differentiate the content (hard code rather than props). Otherwise we'll be passing paragraphs into the components as props which is also OK I guess but feels weird to me - let me know if you prefer one or the other means and I'll make another commit with that update shortly

File: client/src/components/InfoTile.vue [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
         <p class="text-2xl font-bold">{{ props.header }}</p>
         <p class="xs:hidden sm:block">
-            Lorem ipsum dolor sit, amet consectetur 
-            adipisicing elit. Numquam ex blanditiis at, qui deleniti, nostrum 
-            nihil autem vel sequi voluptatum, fuga voluptate reiciendis! Ullam 
-            ipsum corporis eveniet possimus quam ex.
+            Learn more about coding with our highly qualified instructors. <a href="#">EK.edu</a> is oriented around
+            bootcamp graduates, CS graduates, and others who are interested in becoming artisans of software. We aim to 
+            take learners beyond the basics of how to write code, and focus on transforming into an artist of programming.
+
```
Comment (JamesOrson) [inline client/src/components/InfoTile.vue:8]: Nit: Remove empty line

File: client/src/components/Instructors.vue [hunk 2]
```diff
@@ -5,25 +5,23 @@
...
                 <div class="ml-4 xs:my-3 ">
                     <p class="text-virgil">James Orson</p>
-                    <p class="text-virgil xs:hidden sm:block">Lorem ipsum dolor sit amet consectetur adipisicing elit.
-                        Consectetur illum ducimus, deserunt ad, ab a quaerat aspernatur
-                        dolore perferendis accusamus vel iure placeat distinctio odit
-                        quisquam voluptatibus eaque provident, voluptates et numquam
-                        iusto. Ratione, perspiciatis quis doloremque recusandae sit
-                        consequatur eveniet minima laudantium repudiandae placeat
-                        natus suscipit qui necessitatibus iste.</p>
+                    <p class="text-virgil xs:hidden sm:block">James is a chief Site Reliability Engineer at Pure Storage Solutions
```
Comment (JamesOrson) [inline client/src/components/Instructors.vue:8]: Just call it `Pure Storage`

File: client/src/components/Instructors.vue [hunk 3]
```diff
@@ -5,25 +5,23 @@
...
                     <p class="text-virgil">James Orson</p>
-                    <p class="text-virgil xs:hidden sm:block">Lorem ipsum dolor sit amet consectetur adipisicing elit.
-                        Consectetur illum ducimus, deserunt ad, ab a quaerat aspernatur
-                        dolore perferendis accusamus vel iure placeat distinctio odit
-                        quisquam voluptatibus eaque provident, voluptates et numquam
-                        iusto. Ratione, perspiciatis quis doloremque recusandae sit
-                        consequatur eveniet minima laudantium repudiandae placeat
-                        natus suscipit qui necessitatibus iste.</p>
+                    <p class="text-virgil xs:hidden sm:block">James is a chief Site Reliability Engineer at Pure Storage Solutions 
+                        in San Jose, California. He and his wife love their daughter, eating healthy, and participating
```
Comment (JamesOrson) [inline client/src/components/Instructors.vue:9]: `Mountain View, California`

**Review by JamesOrson — CHANGES_REQUESTED** [review]

**Comment by brandonapol** [comment]
sorry for commit name, will improve my vim skills

File: client/src/components/InfoTile.vue [hunk 4]
```diff
@@ -2,10 +2,9 @@
     <div class="relative rounded-lg my-10 p-6 m-5 bg-opacity-50 backdrop-filter backdrop-blur-lg bg-white shadow-xl xs:flex xs:flex-row sm:flex-col flex-col">
         <p class="text-2xl font-bold">{{ props.header }}</p>
         <p class="xs:hidden sm:block">
-            Lorem ipsum dolor sit, amet consectetur 
-            adipisicing elit. Numquam ex blanditiis at, qui deleniti, nostrum 
-            nihil autem vel sequi voluptatum, fuga voluptate reiciendis! Ullam 
-            ipsum corporis eveniet possimus quam ex.
+            Learn more about coding with our highly qualified instructors. <a href="#">EK.edu</a> is oriented around
```
Comment (JamesOrson) [inline client/src/components/InfoTile.vue:5]: What exactly is this `href` trying to point to?

If you want the home page, it should be `/` not `#`

**Review by JamesOrson — CHANGES_REQUESTED** [review]

**Review by JamesOrson — APPROVED** [review]

**Pull request merged**